src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1478,13 +1479,28 @@ public Block get(EnchantItemEvent e) {
 				return e.getEnchantBlock();
 			}
 		}, 0);
+		// Jump Events
 		EventValues.registerEventValue(HorseJumpEvent.class, Entity.class, new Getter<Entity, HorseJumpEvent>() {
 			@Nullable
 			@Override
 			public Entity get(HorseJumpEvent evt) {
 				return evt.getEntity();
 			}
 		}, 0);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Player.class, new Getter<Player, PlayerEvent>() {
```

Comment:
Actually never mind this. Since the `PlayerStatisticIncrementEvent` extends `PlayerEvent` those event values are automatically registered. Just remove this one and the one below it entirely. I'll resolve the ones related to this but keep this review unresolved till changes are committed